charts/mc-router/templates/clusterrole.yaml

```diff
@@ -8,7 +8,7 @@ rules:
   - apiGroups: [""]
     resources: ["services"]
     verbs: ["watch","list"]
-  {{- if .Values.minecraftRouter.autoScaleUp.enabled }}
+  {{- if .Values.minecraftRouter.autoScale.up.enabled }}
```

Comment:
Unfortunately this would be a breaking change for existing chart users. You could do this as an or-expression to allow for both old and new value identifiers.